legacy-bios/mbr.s

```diff
@@ -0,0 +1,28 @@
+        .code16
+        .section .text
+        .globl start
+
+start:
```

Comment:
Usually it's good practice to clear the code-segment as the first thing in bootloaders, you can't trust that the code-segment is valid. Adding the following lines is enough:

```
start:
	jmp 0x0:entry

entry:
```

Reply:
Thank you. That was `jmp $0, $entry`.